eiffel-vocabulary/EiffelConfidenceLevelModifiedEvent.md

```diff
@@ -0,0 +1,41 @@
+# EiffelConfidenceLevelModifiedEvent
+The EiffelConfidenceLevelModifiedEvent declares that an entity has achieved (or failed to achieve) a certain level of confidence, or in a broader sense to tag it. This is particularly useful for promoting various engineering artifacts, such as product revisions, through the continuous integration and delivery pipelin.
+
+Confidence levels may operate at high or low levels of abstraction - ranging from "smokeTestsOk" to "releasable" or "released" - and they may group other confidence levels of lower abstraction levels. They may also be general or very niched, e.g. "releasable" or "reseabableToCustomerX". Confidence levels frequently figure in automated delivery interfaces within a tiered system context: lower level tiers issue an agreed confidence level signaling that a new version is ready for integration in a higher level tier.
+
+## Data Members
+### data.name
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The name of the confidence level. It is recommended for confidence level names to conform with camelCase formatting, in line with the format of key names of the Eiffel protocol as a whole.
+
+### data.value
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ SUCCESS, FAILURE  
```

Comment:
For legal values section we can use JsonSchema enum if we can define a complete list.